boards/nordic/bm_nrf54l15dk/bm_nrf54l15dk_nrf54l051015_oscillator.dtsi

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&hfxo {
+	/* High-frequency external oscillator (HFXO) is required. Status must be "okay". */
+	status = "okay";
+
+	/* Type of load-capacitors to use for the HFXO.
+	 * Options:
+	 *     "internal"  Use internal load-capacitors.
+	 *     "external"  Use external load-capacitors.
+	 */
+	load-capacitors = "internal";
+
+	/* Internal load-capacitance in fF units.
+	 * Only applicable if internal load-capacitors are used for the HFXO.
+	 *
+	 * Default capacitance for the nRF54L15 DK is 15 pF.
+	 */
+	load-capacitance-femtofarad = <15000>;
+};
+
+&lfxo {
+	/* If low-frequency external oscillator (LFXO) should be used.
+	 * Options:
+	 *     "okay"        Use external oscillator (requires that HW layout has XO).
+	 *     "disabled"    Do not use external oscillator (use internal RC).
+	 */
+	status = "okay";
+
+	/* Type of load-capacitors to use for the LFXO.
+	 * Only applicable if using LFXO (status = "okay").
+	 * Options:
+	 *     "internal"  Use internal load-capacitors.
+	 *     "external"  Use external load-capacitors.
+	 */
+	load-capacitors = "internal";
+
+	/* Internal load-capacitance in fF units.
+	 * Not applicable if external load-capacitors are used for the LFXO.
+	 *
+	 * Default capacitance for the nRF54L15 DK is 17 pF.
+	 */
+	load-capacitance-femtofarad = <17000>;
+};
+
+&grtc {
+	owned-channels = <0 1 2 3 4 5 6 7 8 9 10 11>;
+	/* Channels 7-11 reserved for Zero Latency IRQs, 3-4 for FLPR */
```

Comment:
Should we modify this one to match Bare Metal/ SoftDevice use case?
- SoftDevice (MPSL) will use channel 7-11 (i.e. set as child-owned-channel)
- Update comment to reflect this (replace Zero Latency IRQ)
- We don't support FLPR in BM, i.e. should we remove channel 3 and 4 from "child-owned-channel"?

Reply:
I have added a commit to modify this. Please have a look.